tools/api-markdown-documenter/src/api-item-transforms/helpers/TableHelpers.ts

```diff
@@ -221,8 +205,9 @@ export function createDefaultSummaryTable(
 		return undefined;
 	}
 
-	// Only display "Alerts" column if there are any deprecated or alpha/beta items in the list.
-	const hasAlerts = doItemsContainAlerts(apiItems);
+	// Only display "Alerts" column if there are any alerts to display.
```

Comment:
The volume of changes required in this file have convinced me that a unified table builder function is warranted here. But I will do that in a future PR.